src/scripts/main.js

```diff
@@ -1 +1,45 @@
 'use strict';
+
+function toggleScroll() {
+  const body = document.body;
+  const menuComputedStyle = window.getComputedStyle(menu);
```

Comment:
Variables 'menu' and 'product' are used before they are defined. It's best to define your variables at the top of your scope to avoid potential ReferenceErrors or unexpected behavior.